src/server/scripts/Spells/spell_item.cpp

```diff
@@ -2831,6 +2831,76 @@ class spell_item_toy_train_set_pulse : public SpellScriptLoader
     }
 };
 
+enum DeathChoiceSpells
+{
+    SPELL_DEATH_CHOICE_NORMAL_AURA      = 67702,
+    SPELL_DEATH_CHOICE_HEROIC_AURA      = 67771,
+    SPELL_DEATH_CHOICE_NORMAL_STRENGHT  = 67708,
+    SPELL_DEATH_CHOICE_NORMAL_AGILITY   = 67703,
+    SPELL_DEATH_CHOICE_HEROIC_STRENGHT  = 67773,
+    SPELL_DEATH_CHOICE_HEROIC_AGILITY   = 67772
```

Comment:
First off, all your work so far... Priceless...

To get back on topic at hand, although being purely cosmetic, why not order the enums either by type or difficulty?

```
SPELL_DEATH_CHOICE_NORMAL_AURA      = 67702,
SPELL_DEATH_CHOICE_NORMAL_STRENGHT  = 67708,
SPELL_DEATH_CHOICE_NORMAL_AGILITY   = 67703,
SPELL_DEATH_CHOICE_HEROIC_AURA      = 67771,
SPELL_DEATH_CHOICE_HEROIC_STRENGHT  = 67773,
SPELL_DEATH_CHOICE_HEROIC_AGILITY   = 67772
```

or
```
SPELL_DEATH_CHOICE_NORMAL_AURA      = 67702,
SPELL_DEATH_CHOICE_HEROIC_AURA      = 67771,
SPELL_DEATH_CHOICE_NORMAL_STRENGHT  = 67708,
SPELL_DEATH_CHOICE_HEROIC_STRENGHT  = 67773,
SPELL_DEATH_CHOICE_NORMAL_AGILITY   = 67703,
SPELL_DEATH_CHOICE_HEROIC_AGILITY   = 67772
```
?

Also, I believe correct spelling is STREN**GTH**, not STRENGHT.